cs/libdpr/samples/DprCounters/DprCounters/python/start_cluster.py

```diff
@@ -0,0 +1,250 @@
+from ctypes import ArgumentError
+from http import server
+from posixpath import basename
+import resource
+from kubernetes import client, config
+import os
+import time
+from numpy import void
+
+import yaml
+
+
+class KubernetesCluster():
+
+    DEFAULT_STORAGE = "1Gi"
+    SUPPORTED_SERVER_TYPES = ["counter"]
+    BASE_PORT = 6380
+    BASE_AZURE_PORT = 31101
+    LOCAL_DIRECTORY = "/mnt/c/Users/cetko/OneDrive/Desktop/MEng Thesis/new_faster/cs/libdpr/samples/DprCounters/DprCounters"
```

Comment:
Can you put some of these into a config class that is taken in at initialization?